docs/getting_started/components/hls.md

```diff
@@ -13,4 +13,8 @@ Currently, there are no environment variables related to this component.
 
 ## Example Docker commands
 
-Your docker commands shouldn't be affected by adding this component.
\ No newline at end of file
+The HLS playlist will be created inside the Docker container. To access it from the host,
```

Comment:
Unfortunately, this won't work on Linux. On Linux, the created host directory will have root owner as docker runs with root privilages. On the other hand, Jellyfish runs as nobody and won't be able to write to the root directory.

Not sure what we want to do with this.

In general, there are a couple of solutions:

* use named volumes - but at least in my case, I don't know how to get data out of the named volume
* run docker as root, setup privilages in docker entrypoint and continue running as non-root user using tool like gosu. That's how redis  do this
* try to prepare volume directory before running docker - I didn't mange to make it working